lib/engine/game/g_1847_ae/step/draft.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1847AE
+      module Step
+        class Draft < Engine::Step::Base
```

Comment:
If you extend `Engine::Step::SimpleDraft` instead of `Engine::Step::Base` you get several of these methods defined for you.